pcm.h

```diff
@@ -1,24 +1,26 @@
+#include <stdint.h>
+
 class PcmOutput {
 public:
     virtual ~PcmOutput() {}
-    virtual int getSampleRate() = 0;
+    virtual uint32_t getSampleRate() = 0;
     virtual void output(const short *buf, int n) = 0;
     virtual void flush() {}
 };
 
-PcmOutput *makePcmOutputWav(const char *fn);
+PcmOutput *makePcmOutputWav(const char *fn, uint32_t rate);
 
 #ifdef unix
-PcmOutput *makePcmOutputUnix();
-#define makePcmOutputAudio makePcmOutputUnix
+PcmOutput *makePcmOutputUnix(uint32_t rate);
+#define makePcmOutputAudio(rate) makePcmOutputUnix(rate)
```

Comment:
It's not necessary to include the parameter `(rate)` here. The macro works fine without mentioning the parameter list. Recommend not changing the source line if you don't need to.